### ISSUE TITLE
Wrong client stats

### ISSUE THREAD

**Issue by benbro** [reith/udpstress#2]
If there is packet loss between server->client, the server show higher send and receive numbers then the client. This doesn't make sense.

The client should update sent_pkts and sent_size when sending and not when getting ack.
It should also decrease not_acked_pkts by 1.
I will also be useful to see nack_pkts in the report.
https://github.com/reith/udpstress/blob/master/src/genserver_client.erl#L83
https://github.com/reith/udpstress/blob/master/src/genserver_client.erl#L93

**Comment by reith** [comment]
I agree report was semantically different between server and client. I used to swap client and server in each node, so having `sent` report as `actually sent` had advantage for me, yet it's confusing. I think #3 resolved this.

**Comment by benbro** [comment]
Thank you. I'll test it.

When running the test between two VMs on Google Cloud the outbound number packets of the client and the inbound of the server was double than what I asked udpstress to send. The outbound of the server and inbound of the client were fine. udpsterss opens a socket on all interfaces (port 0) and I was using the server private IP address. Do you know why the packet rate reported is twice of what it should be?
